pkg/gce-pd-csi-driver/controller.go

```diff
@@ -361,7 +361,7 @@ func (gceCS *GCEControllerServer) createVolumeInternal(ctx context.Context, req
 	if _, err := getMultiWriterFromCapabilities(volumeCapabilities); err != nil {
 		return nil, status.Errorf(codes.InvalidArgument, "VolumeCapabilities is invalid: %v", err.Error())
 	}
-	err = validateStoragePools(req, params, gceCS.CloudProvider.GetDefaultProject())
+	err = validateStoragePools(req, params, gceCS.CloudProvider.GetDefaultProject(), gceCS.enableHdHA)
```

Comment:
please add a unit test for this to TestValidateStoragePools

Reply:
This is an open source repo, so we have a lot of contributors who are unaware of previous  features. This will help prevent someone who is unaware of your feature, from breaking the functionality.

Reply:
Makes sense. Thanks